src/components/pages/order/OrderPage.js

```diff
@@ -1,20 +1,36 @@
+import { useState } from 'react';
 import styled from 'styled-components';
+import OrderContext from '../../../context/OrderContext';
 import { theme } from '../../../theme';
 import Main from './main/Main';
 import Navbar from './navbar/Navbar';
 
 export default function OrderPage(){
   // state 
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+  const [isCollapsed, setIsCollapsed] = useState(false)
+  const [currentSelectedTab, setCurrentSelectedTab] = useState("add")
   // comportements
+  const orderContextValue = {
+    isModeAdmin,
+    setIsModeAdmin,
+    isCollapsed,
+    setIsCollapsed,
+    currentSelectedTab,
```

Comment:
```suggestion
    currentTabSelected,
```
plutôt dans ce sens 😄